packages/react-instantsearch-hooks/src/components/DynamicWidgets.tsx

```diff
@@ -26,9 +27,16 @@ export type DynamicWidgetsProps = Omit<
 
 export function DynamicWidgets({
   children,
-  fallbackComponent: Fallback = FallbackComponent,
+  fallbackComponent: Fallback = DefaultFallbackComponent,
   ...props
 }: DynamicWidgetsProps) {
+  const FallbackComponent = React.useRef(Fallback);
+
+  warn(
+    Fallback === FallbackComponent.current,
+    'The `fallbackComponent` prop of `DynamicWidgets` changed between renders. Please provide a stable reference.'
+  );
```

Comment:
Would adding a link to the doc and update it to show an example of how to achieve that help?

I'm thinking of this part: https://www.algolia.com/doc/api-reference/widgets/dynamic-facets/react-hooks/#widget-param-fallbackcomponent.

Reply:
is 0f6c9fe clear?